test/infrastructure/docker/config/default/kustomization.yaml

```diff
@@ -11,14 +11,36 @@ namespace: capd-system
 namePrefix: capd-
 
 # Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
+commonLabels:
+  clusterctl.cluster.x-k8s.io: ""
+  clusterctl.cluster.x-k8s.io/provider: "docker"
 
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in crd/kustomization.yaml
 #- ../webhook
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
 #- ../certmanager
 
+patchesJson6902:
+- target:
+    group: apps
+    version: v1
+    kind: Deployment
+    name: controller-manager
+    namespace: system
+  path: deployment_remove_selectors.yaml
+- target:
+    version: v1
+    kind: Service
+    name: controller-manager-metrics-service
+    namespace: system
+  path: service_remove_selectors.yaml
+# - target:
```

Comment:
Remove commented lines?

Reply:
Ah! I meant to add this line above it `# [WEBHOOK] ...` because if the user does enable `../webhook` above, I want the user to be aware this is required so that the webhook service's selectors does not get modified with the clusterctl labels.